src/near_ext.rs

```diff
@@ -259,18 +259,26 @@ impl<'a> vm::Ext for NearExt<'a> {
     }
 
     /// Creates log entry with given topics and data
-    fn log(&mut self, _topics: Vec<H256>, data: &[u8]) -> EvmResult<()> {
+    fn log(&mut self, topics: Vec<H256>, data: &[u8]) -> EvmResult<()> {
```

Comment:
where does this function get called? I was looking at it the other day and couldn't find where

Reply:
It's called by the parity evm interpreter whenever a log opcode is executed